app/views/ReactionsView/index.ts

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+import { connect } from 'react-redux';
+import { StackNavigationProp } from '@react-navigation/stack';
+import { RouteProp } from '@react-navigation/core';
+import { withSafeAreaInsets } from 'react-native-safe-area-context';
+import ScrollableTabView, { ScrollableTabBar } from 'react-native-scrollable-tab-view';
+
+import I18n from '../../i18n';
+import { withTheme } from '../../theme';
+import { themes } from '../../constants/colors';
+import { withDimensions } from '../../dimensions';
+import SafeAreaView from '../../containers/SafeAreaView';
+import Emoji from '../../containers/message/Emoji';
+
+import sharedStyles from './styles';
+import TabPage from './TabPage';
+
+interface IReactionsViewProps {
+	navigation: StackNavigationProp<any, 'ReactionsView'>;
+	theme: string;
+	baseUrl: string;
+	customEmojis: any;
+	route: RouteProp<{ ReactionsView : { reactions: any } }, 'ReactionsView'>;
+}
+
+interface IReactionsLabelProps {
+	reactions: any;
+	baseUrl: string;
+	getCustomEmoji: Function;
+	name: string;
+	page: number;
+	theme: string;
+}
+
+class TabLabel extends React.PureComponent<IReactionsLabelProps> {
+	constructor(props: IReactionsLabelProps) {
+		super(props);
+	}
+
+	render = () => {
+		const { name, baseUrl, getCustomEmoji, reactions, page, theme } = this.props;
+
+		return (
+			<View style={ sharedStyles.tabView }>
```

Comment:
@diegolmello The added JSX is failing for some reason. Just can't figure out why. Any idea?

Reply:
Nevermind. Found the fix. Just had to rename the files to .tsx